jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -35,22 +35,15 @@
     "vue-upload-component": "^3.1.2"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.19.1",
-    "@babel/register": "^7.18.9",
```

Comment:
I think it should be fine to drop support to older browsers (IE, older Ffox, Chrome... Ffox and Chrome have supported newer JS for years now, nobody should be using IE... this reduces our build time & size - makes UI/JS development a lot easier too).